common-library/pom.xml

```diff
@@ -19,6 +19,7 @@
         <sonar.organization>nashtech-garage</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
         <sonar.projectKey>nashtech-garage_yas-common-library</sonar.projectKey>
+        <org.keycloak.version>16.1.0</org.keycloak.version>
```

Comment:
Can you explain the reason for not using the same version as the parent's pom.xml?